CHANGES.rst

```diff
@@ -110,6 +110,9 @@ Bugs fixed
 * #12916: Restore support for custom templates named with the legacy ``_t``
   suffix during ``apidoc`` RST rendering (regression in 7.4.0).
   Patch by James Addison.
+* #12451: Add further preconditions before copyright notice year substitution
+  occurs.
+  Patch by James Addison.
```

Comment:
Perhaps?: "further preconditions" is admirably vague!
```suggestion
* #12451: Only substitute copyright notice years with values from ``SOURCE_DATE_EPOCH``
  when the copyright year is the current year
  and ``SOURCE_DATE_EPOCH`` is not in the future.
  Patch by James Addison and Adam Turner.
```

Reply:
Yep, definitely an improvement.  I think my preferred phrasing is now to mention the phrase 'system-clock year' when explaining the year-matching logic.  I'll push a commit that combines your suggestion with that to the branch soon.